### ISSUE TITLE
[Quest][Classic] The Tome of Divinity

### ISSUE THREAD

**Issue by k10431421** [dalaranwow/dalaran-wow#1565]
"Take the Symbol of Life and resurrect Henze Faulk in Elwynn."

After casting the Symbol of Life on Henze, it is consumed without quest completion. Upon attempting to abandon and attempt again, a new symbol of life is not given.

**Comment by lightstylez** [comment]
Still an issue. I think I did the quest when I was low lvl. I rezzed him but the symbol disappeared and quest didn't complete. After going back to stormwind no matter how many times I cancel/reaccept the quest or relog he will not give me another symbol and so I am playing without the resurrect skill which is really bad when a bunch of ppl die and I'm playing the only class who can supposedly rez and I have to explain to them why I can't forcing them to spend ages running back. I've sent in a ticket so hopefully this can be fixed manually by a GM until the quest gets fixed.
